src/azure-openapi-validator/tests/composite-azure-test.ts

```diff
@@ -0,0 +1,22 @@
+import { Message } from '../../jsonrpc/types';
+import { suite, test, slow, timeout, skip, only } from "mocha-typescript";
+import { safeLoad } from "js-yaml";
+import { AutoRestPluginHost } from "../../jsonrpc/plugin-host";
+import { run } from "../../azure-openapi-validator";
+import { ReadFileAsString, CollectTestMessagesFromValidator } from "./utilities/tests-helper";
+import * as assert from "assert";
+
+
+@suite class CompositeAzureTest {
+  @test @timeout(120000) async "description should not be parameter name"() {
+    const file = 'src/azure-openapi-validator/tests/resources/incomplete-descriptions.json';
+    const openapiDefinitionDocument = ReadFileAsString(file);
+    const openapiDefinitionObject = safeLoad(openapiDefinitionDocument);
+    let messages: Message[] = await CollectTestMessagesFromValidator(openapiDefinitionObject);
+
+    let msgs = messages.filter((msg) => {
```

Comment:
We need mechanism like `AssertExtensions` 

Reference: https://github.com/Azure/autorest/blob/master/src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs#L17